CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2018 Jonathan Müller <jonathanmueller.dev@gmail.com>
```

Comment:
I don't think I can claim this in my own project. Can you please remove this notice?

Reply:
@LiraNuna my bad, updated